components/02-elements/form/input/input--textarea.hbs

```diff
@@ -1,14 +1,14 @@
-<div class="input {{ class }}" {{{ attribute }}}>
+<div class="input {{ class }}" {{ attribute }}>
     <label class="input__label {{ label.class }}"
            for="textarea"
-           {{{ label.attribute }}}
+           {{ label.attribute }}
     >
         {{ label.text }}
     </label>
 
-    <textarea class="input__field input__field--textarea"
+    <textarea class="input__field input__field--textarea {{ field.class }}"
               id="textarea"
```

Comment:
move to config (in simple input too)
you can add `name` attribute :)

Reply:
done

Reply:
done